content/XX.Peptide Fragmentation

```diff
@@ -0,0 +1,65 @@
+Collisions in beamCID typical impart enough internal energy to fragment peptide ions with a single collision, although the beam-type nature of rapidly moving ions into the collision cell can generate multiple collision events as ion traverse the device.
+Because the increase in internal energy happens rapidly before energy can be redistributed, beamCID can generate fragments that are not necessarily derived from cleavage of the most labile bonds (e.g., PTM-modified peptides, discussed below), but spectra are often dominated by b/y-type ions from amide bond cleavage.
+BeamCID can also generate secondary fragments, such as immonium ions from side chain losses [@DOI:10.1021/ac8006076] or a-type fragment ions that come from water loss from b-type ions due to multiple collision events (note: a-type ions can form as primary fragmentation products in other dissociation methods).
+The simplicity of beamCID, which requires simply an rf-only collision cell, has made it widely implemented on most instrument platforms used in modern proteomics.
+
+A second form of CID is called resonant CID (resCID), where the internal energy of peptide ions is slowly increased through multiple low-energy collisions [@DOI:10.1002/(SICI)1096-9888(199705)32:5<461::AID-JMS515>3.0.CO;2-H].
+Here, helium gas is most often used, as it imparts less energy per collision, and activation typically happens in ion trap devices where supplemental frequencies can be used to excite ions.
+In other words, ions are trapped using axial rf-frequencies, and an additional rf-frequency is applied to the electrodes of the ion trap [@DOI:10.1016/S1044-0305(02)00384-7; @DOI:10.1002/mas.21549].
+This supplemental rf is selected to have a frequency resonant with the fundamental frequency of the ions to be fragmented, as determined by the Matthieu equations, which excites the ions of interest so that they have increased kinetic energy as they move in the ion trap.
+The increased kinetic energy creates more collisions with the background helium gas to slowly build up the internal energy of the precursor ions until the dissociation energy of the most labile bond is reached, causing fragmentation [@DOI:10.1016/S0076-6879(05)02005-7].
+Once ions dissociate, the fragments have different m/z values than the precursor ions, meaning they fall out of resonance with the supplemental rf and are no longer activated.
+Thus, resCID typically fragments only the most labile bonds in precursor ions and does not have secondary fragmentation behavior.
+As above, for non-modified peptide ions, this typically generates sequence-informative b- and y-type product ions.
+For modified peptides where the bonds connecting the modification to an amino acid are more labile than peptide bonds, e.g., phosphopeptides and glycopeptides, resCID MS/MS spectra can be dominated by products ions only of the PTM-loss rather than sequence informative fragment ions, although many factors govern this behavior.
```

Comment:
```suggestion
For modified peptides where the bonds connecting the modification to an amino acid are more labile than peptide bonds, e.g., phosphopeptides and glycopeptides, resCID MS/MS spectra can be dominated by products ions only of the PTM-loss rather than sequence informative fragment ions, although many factors govern this behavior [@DOI:10.1021/pr0705136; @DOI:10.1021/ac0497104].
```